src/components/EditableTypography/EditableTypography.tsx

```diff
@@ -109,22 +106,32 @@ const EditableTypography: VibeComponent<EditableTypographyProps, HTMLElement> =
       }
     }, [isEditing]);
 
+    const AnimatedTypography = motion(TypographyComponent);
+    const isOverflowing = useIsOverflowing({ ref: typographyRef, ignoreHeightOverflow: true });
+    const shouldAnimate = useMemo(() => isOverflowing && isEditing !== null, [isEditing, isOverflowing]);
```

Comment:
Don't forget this null handling (it's never null)

Reply:
Oh right thanks! I remembered for a moment and then forgot again :(